map/autoware_pointcloud_map_validator/package.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>autoware_pointcloud_map_validator</name>
+  <version>0.0.0</version>
+  <description>The autoware_pointcloud_map_validator package</description>
+  <maintainer email="yhisaki31@gmail.com">Yukinari Hisaki</maintainer>
+  <maintainer email="satoshi.ota@tier4.jp">Satoshi Ota</maintainer>
+  <maintainer email="kosuke.takeuchi@tier4.jp">Kosuke Takeuchi</maintainer>
+  <maintainer email="shintaro.sakoda@tier4.jp">Shintaro Sakoda</maintainer>
+  <maintainer email="fumiya.watanabe@tier4.jp">Fumiya Watanabe</maintainer>
+  <maintainer email="uken.ryu.2@tier4.jp">Yuxuan Liu</maintainer>
+  <maintainer email="anh.nguyen.2@tier4.jp">Anh Nguyen</maintainer>
+  <maintainer email="masahiro.sakamoto@tier4.jp">Masahiro Sakamoto</maintainer>
+  <maintainer email="taiki.yamada@tier4.jp">Taiki Yamada</maintainer>
+  <maintainer email="yamato.ando@tier4.jp">Yamato Ando</maintainer>
```

Comment:
I would like to add: 
`<maintainer email="kei.nakazato@proxima-ai-tech.com">Kei Nakazato</maintainer>`